docker-compose.yml

```diff
@@ -30,6 +25,7 @@ services:
     <<: *logging-config
     depends_on:
       - fastapi
+      - yovisto
```

Comment:
In the nginx configuration nginx depends on both fastapi and yovisto - see "upstream". Of course, the nginx container would just crash until fastapi and yovisto are there. But that seems rather unhealthy. And yes, if one of them is down or still initializing, it respons with 502.

Reply:
HMM, AFAIK, the nginx container wouldn't crash. Its just a proxy, and indifferent about the availability of the backend services. If some of the backends are not available it would return a 502 or 404 to the user.